tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -146,6 +146,7 @@ module.exports = {
       return this.waitForElementVisible(linkRowDeleteButton)
         .initAjaxCounters()
         .click(linkRowDeleteButton)
+        .click('.oc-modal-body-actions-confirm')
```

Comment:
Please don't use selectors directly in the code here, but create a selector at the bottom of the file and use it here. Afaik you need to wait for the button to be visible after clicking the delete button.